tests/daemon/test-persistent-cache.c

```diff
@@ -954,33 +953,20 @@ static void
 test_persistent_cache_does_not_compute_offset_when_boot_id_is_same (gboolean     *unused,
```

Comment:
Here is my attempt to clarify/correct the documentation for this method. If it doesn't help you understand how this function works, then it clearly isn't doing a very good job.

```
 /* Tests that releasing and recreating the persistent cache within the same boot
  * doesn't change the boot offset.
  *
  * Instantiates a persistent cache with no preexisting boot metadata file,
  * which triggers a reset to a boot offset of 0. Then unrefs the persistent
  * cache, mutates the boot ID in the metadata file, and makes the persistent
  * cache anew to simulate a new boot and trigger computation of a non-zero boot
  * offset. Finally, unrefs and makes anew the persistent cache once more,
  * prompting it to detect the unchanged boot ID and preserve the boot offset.
  */
```